doc/en/user/source/data/webadmin/layerpreview.rst

```diff
@@ -12,17 +12,19 @@ This page provides layer views in various output formats. A layer must be enable
 Each layer row consists of a type, name, title, and available formats for viewing.
 
 .. list-table::
-   :widths: 5 95 
+   :widths: 5 70
    :header-rows: 1
- 
+
    * - Field
      - Description
-   * - .. image:: img/data_layers_type1.png
-     - Raster (grid) layer
-   * - .. image:: img/data_layers_type2.png
-     - Vector (feature) layer  
-   * - .. image:: img/preview_layergroup.png
-     - Layer group 
+   * - .. image:: img/raster_icon.png
+     - Raster (grid)
+   * - .. image:: img/polygon_icon.png
+     - Polygon
+   * - .. image:: img/line_string_icon.png
+     - Line
+   * - .. image:: img/point_icon.png
+     - Point
```

Comment:
If you want this to be complete (not saying that it has to be) then there's also an icon for WMS cascaded layers, and a "generic geometry" one. See https://github.com/geoserver/geoserver/blob/master/src/web/core/src/main/java/org/geoserver/web/CatalogIconFactory.java#L114 and the associated methods